### ISSUE TITLE
Enhancements and Feature Requests

### ISSUE THREAD

**Issue by raineorshine** [tjunnone/npm-check-updates#484]
For housekeeping purposes, enhancements and feature requests can now be found closed under the [Revive Me](https://github.com/tjunnone/npm-check-updates/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3Arevive-me+) tag. I just don't have the time personally. If a PR is started by a generous individual, I am happy to re-open. Thanks!

**Comment by XhmikosR** [comment]
@raineorshine maybe this can be closed now?

**Comment by raineorshine** [comment]
Yes. This is very old, and probably belongs in the README anyway.
